src/sender.js

```diff
@@ -142,13 +150,18 @@ async function main({ msg, ackMsg, nackMsg, sendToQueue, channel }) {
     logger.info(`Finished processing msg`)
```

Comment:
I think the level should `debug` here. Since it is not required to report that the message was processed successfully if no errors appear.

Reply:
Agree, done in 7ca7e58e372fea95bbf56eda1e5bcca4e56f442f.